ChangeLog.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+### 2022-03-XX - 5.14.1
+
+| Library                                                                 | Min Version |
+| ----------------------------------------------------------------------- | ----------- |
+| `@autorest/core`                                                        | `3.6.2`     |
+| `@autorest/modelerfour`                                                 | `4.19.1`    |
+| `azure-core` dep of generated code                                      | `1.20.1`    |
+| `msrest` dep of generated code                                          | `0.6.21`    |
+| `azure-mgmt-core` dep of generated code (If generating mgmt plane code) | `1.3.0`     |
+
+**New Features**
+
+- Support flag `--security`/`--security-scopes`/`--security-header-name` and security definition in swagger #1161
```

Comment:
I think we're trying to push users to use these configs as [configurations in the swagger](https://github.com/Azure/autorest/blob/main/docs/generate/authentication.md#aad-token-authentication) more than flags. This is because we are trying to create a language-agnostic way where reviewers etc can easily look at the swagger and know what the credentials will be for the SDK.

I think the current one highlights the flags more, so I think we should just say "add support for security configurations in the swagger" and link to tim's doc https://github.com/Azure/autorest/blob/main/docs/generate/authentication.md for more information

Reply:
ok